WcaOnRails/app/models/result.rb

```diff
@@ -127,7 +127,7 @@ def compute_correct_average
         100 * sum_moves / counting_solve_times.length
```

Comment:
@jfly this one too needs rounding :(
Don't bother fixing it though, I'll send the "check results" PR later today with the fix included.